app.js

```diff
@@ -0,0 +1,23 @@
+//.image
+console.log('Hello from JS!');
+// The type of the event
+const eventType = 'click';
+// The target of the event
+const myButton = document.querySelector("#randomize");
+// The callback - what do you want to
+//                do as a response?
+const myCallback = function(event) {
+  console.log('The button was clicked!', event);
+
+
```

Comment:
you can make the the bonus by getting the height and width as a variable and using them as a part of the URL

```

let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```